### PR TITLE
initial post

### PR DESCRIPTION
**Pull request by peterdekinder**
### Thank you for your contribution! Please go through this small checklist to prepare your pull request to be reviewed as soon as possible :)
* [ ] All authors were added to `_data/authors.yml` with a picture and an up-to-date bio
* [ ] The blog post image is of ratio 500x293
  * Feel free to leave a comment if you don't know how to resize your image and someone will gladly help you out
* [ ] The [**one sentence per line** method](https://raw.githubusercontent.com/brandon-rhodes/blog/master/texts/brandon/2012/one-sentence-per-line.rst) has been adopted
* [ ] A spell check has been done
* [ ] All links open in a new tab
  * Example: `[Amazing tech blog](https://ordina-jworks.github.io){:target="_blank" rel="noopener noreferrer"}`
* [ ] The post's content properly resizes according to the window's size
  * Tip: Just shorten your window's horizontal length to easily test this
  * To have images properly resize use `class="image fit"`
  * To have YouTube videos properly resize, wrap the iframe into a `<div class="responsive-video">`
* [ ] If a table of content is: check whether all links are working correctly

### Any specific remarks for the reviewers or anything you need help with?
\<enter text here>

### Checklist for the reviewers before merging
* [ ] Publishing date of the post has been correctly set
  * This can be done by renaming the filename or by setting the date in the header section of the post

### CODE REVIEW

File: _posts/2022-06-13-connecting-the-pods.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+The first chapters go about comparing the organization to a network, where significant events can travel the different nodes of a network and have an impact on all of them. 
+The organization of today is a participant in a multitude of social networks (employees, customers, competitors, shareholders...), and each of these shapes how the power balance within the organization is established.
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-06-13-connecting-the-pods/networkanatomy.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+Traditional companies have an information stream that is pretty much one direction. 
+The company would have an unending stream of products that would be shipped out to customers, and so the company is organized and optimized for such an exchange. 
+However, this industrial age has given way to a situation where the feedback from these customers will change how the company goes about things and forces companies to create customer relationships where they will have to listen to these customers.
```
Comment (pietervincken) [inline _posts/2022-06-13-connecting-the-pods.md:33]:
```suggestion
However, this industrial age has given way to a situation where the feedback from these customers will change how the company goes about things. 
It forces companies to create customer relationships where they will have to listen to these customers. 
```

File: _posts/2022-06-13-connecting-the-pods.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+These distractions can range from a multitude of new opportunities to internal efficiency drives, over-expansion (with product dilution as a result), and so on. 
+This can also lead to an automation of processes to handle the increasing complexity with the caveat of these automations becoming too restrictive, and limiting the creativity and adaptability of your services. 
+Where the author argues this case, I would however stipulate that over-automation is easily mitigated by determining which of your processes are run-of-the-mill and which add value to your unique selling point or differentiator. 
+The former are ideal candidates for such an automation drive (or standardization) whereas the latter are more suited for custom development that allows for easy adaptation. 
+This comes down to finding a balance between efficiency and adaptability/innovation.
+
+Another game changer is the number of companies that go belly up. 
+This is largely due to the complexity of the market increasing, with new technologies popping up all over and disrupting the market in addition to the previously mentioned globalization increases the number of competitors companies have to deal with.
```
Comment (ChrisDeBruyne) [inline _posts/2022-06-13-connecting-the-pods.md:70]: This sentence looks a bit odd, does it miss a comma ? I'm not sure what it is but something seems off.

File: _posts/2022-06-13-connecting-the-pods.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+{: refdef}
+
+Traditional companies have an information stream that is pretty much one direction. 
+The company would have an unending stream of products that would be shipped out to customers, and so the company is organized and optimized for such an exchange. 
+However, this industrial age has given way to a situation where the feedback from these customers will change how the company goes about things and forces companies to create customer relationships where they will have to listen to these customers. 
+This service economy necessitates new habits, behaviors, and organizational structures. 
+
+The author states that the change towards this service economy is supported by three factors: product saturation, information technology, and urbanization. 
+There are some issues with this analysis as far as I’m concerned. 
+Or rather some tweaks I would add with the benefit of ten years of hindsight.
```
Comment (pietervincken) [inline _posts/2022-06-13-connecting-the-pods.md:38]:
```suggestion
Or rather, I woud add some tweaks with the benefit of ten years of hindsight. 
```

File: _posts/2022-06-13-connecting-the-pods.md [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+{: refdef}
+
+## Part 3: How does a Connected Company work?
+
+Business agility is the name of the game and the Connected Company tries to address this topic through the standard technological ways that are available in an attempt to make the company more built for agility:
+* Agile Development with small autonomous teams
+* Service Oriented Development with service contracts, a focus on composability, and loose coupling
+
+The idea is to transform a business into a podular organization. 
+This is an approach to combines the traditional organization split into divisions with pods where you divide the required work into “businesses within businesses”, which can function independently from other pods, and take on the responsibility for a complete “service”.
```
Comment (ChrisDeBruyne) [inline _posts/2022-06-13-connecting-the-pods.md:145]:
```suggestion
This is an approach that combines the traditional organization split into divisions with pods where you divide the required work into “businesses within businesses”, which can function independently from other pods, and take on the responsibility for a complete “service”. 
```

File: _posts/2022-06-13-connecting-the-pods.md [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+The connected company takes actions, which leads to feedback and discovery based on the outcome of such actions. 
+Once these are collected, you reflect upon how to change the way actions are taken to improve their result. 
+When compared to more traditional ways of learning such as the Double-Loop Learning, to me, this seems like the reflection part is portrayed as a bit too straightforward. 
+If the results of your actions are the only thing to reflect upon, the reframing of the mindset is missed.
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-06-13-connecting-the-pods/learning.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+To go a bit more into detail on this type of learning: Professor Chris Argyris of Harvard Business School identifies three possible levels of learning in an organization, which he coined single-loop, double-loop, and deutero learning.
```
Comment (ChrisDeBruyne) [inline _posts/2022-06-13-connecting-the-pods.md:190]: link or reference to this?

File: _posts/2022-06-13-connecting-the-pods.md (hunk 2 above)
Comment (pietervincken) [inline _posts/2022-06-13-connecting-the-pods.md:70]:
```suggestion
This is largely due to the increased complexity of the market, with new technologies popping up all over and disrupting the market. 
This in addition to the previously mentioned globalization increases the number of competitors companies have to deal with. 
```

File: _posts/2022-06-13-connecting-the-pods.md [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+So what is a pod? 
+A small, autonomous unit that has the mandate and the capabilities to deliver whatever it is that its customers value. 
+It is designed to take decisions and implement changes as quickly and as close to the customer as possible. 
+Whereas the traditional company employs processes that can be seen as recipes or a series of steps that need to be executed, the podular organization is structured as a net, distributing the workload based on goals instead of allocating individual steps or stages. 
+And the pods are not necessarily part of the same organization or company but could extend across several to couple its inherent benefits with more economy-of-scale oriented benefits.
+
+This gives a number of advantages based on its lattice-like structure, exchanging consistency and efficiency for flexibility and creativity:
+* Pods are flexible: changing the structure and purpose is easy and fast.
+* Pods are allowed to fail as the network will provide redundancy by adjusting the purpose of similar pods.
+* Pods can scale up fast by adding additional pods and letting them immediately contribute.
```
Comment (pietervincken) [inline _posts/2022-06-13-connecting-the-pods.md:170]: Are we still talking about businesses and not distributed systems? 😄 I love this!

**Review by pietervincken — COMMENTED** [review]
LGTM. Comments are just suggestions.

File: _posts/2022-06-13-connecting-the-pods.md (hunk 2 above)
Comment (peterdekinder) [reply _posts/2022-06-13-connecting-the-pods.md:70]: fixed

File: _posts/2022-06-13-connecting-the-pods.md (hunk 4 above)
Comment (peterdekinder) [reply _posts/2022-06-13-connecting-the-pods.md:145]: fixed

File: _posts/2022-06-13-connecting-the-pods.md [hunk 7]
```diff
@@ -0,0 +1,291 @@
...
+The balanced application of both of these levels is called deutero learning, which is the preferred manner of proceeding. 
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-06-13-connecting-the-pods/deutero.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+The successive approximation approach in the book focuses on three levels of learning: 
+How entrepreneurs learn, how companies learn, and how the platform learns.
+
+The entrepreneurial method for learning (as stated by Saras Sarasvathy) states that entrepreneurs focus on the qualities they possess within the organization and attempt to utilize these to their greatest effect.
```
[the remark this answers is not in the document]
Comment (peterdekinder) [reply _posts/2022-06-13-connecting-the-pods.md:207]: no particular source. She mentions it in a lot of her work

File: _posts/2022-06-13-connecting-the-pods.md (hunk 5 above)
Comment (peterdekinder) [reply _posts/2022-06-13-connecting-the-pods.md:190]: same. no particular source. He mentions it in a lot of his work

File: _posts/2022-06-13-connecting-the-pods.md (hunk 3 above)
Comment (peterdekinder) [reply _posts/2022-06-13-connecting-the-pods.md:38]: yes, stylistic choice

File: _posts/2022-06-13-connecting-the-pods.md [hunk 8]
```diff
@@ -0,0 +1,291 @@
...
+The author will propose the Connected Company approach for this.
+
+Where the Connected Company wants to make a paradigm shift, is to move the focus from procedures and rules where there is the idea of an optimal solution that balances the needs of the company with those of the customers towards a company putting the focus more on the customer and to become more like a support system for those customers. 
+Ashby’s Law (also called the Law of Requisite Variety) is mentioned in one breath with this view: Any control system must be capable of variety that is greater or equal to the variety in the system to be controlled. 
+However, this seems to assume that processes within companies are fixed, while the whole discipline of Business Process Management is predicated on the fact that a process is never finished, and that there is always room for improvement, either induced by internal or external factors.
+
+In any case, a case is made for customers resisting standardization, and that to meet such required variety, the company needs innovation. 
+And one of those ways to achieve this is to give the employees of the company the freedom to experiment.
```
[the remark this answers is not in the document]
Comment (peterdekinder) [reply _posts/2022-06-13-connecting-the-pods.md:100]: fixed

File: _posts/2022-06-13-connecting-the-pods.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (peterdekinder) [reply _posts/2022-06-13-connecting-the-pods.md:145]: fixed

File: _posts/2022-06-13-connecting-the-pods.md (hunk 6 above)
Comment (peterdekinder) [reply _posts/2022-06-13-connecting-the-pods.md:170]: :)

File: _posts/2022-06-13-connecting-the-pods.md (hunk 1 above)
Comment (peterdekinder) [reply _posts/2022-06-13-connecting-the-pods.md:33]: stylistic choice :)

**Pull request merged**